makejdk.sh

```diff
@@ -47,6 +47,8 @@ TARGET_DIR=""
 BRANCH=""
 KEEP=false
 JTREG=false
+JVM_VARIANT=""
+ALTERNATE_VARIANT=""
```

Comment:
_Minor comment:_  I'm not wild about this name.  If we are having `<something>_VARIANT` then the `<something>` should be a noun.  Consider `BUILD_VARIANT`?  Just a bit more descriptive.

Reply:
I admit, the 8 milliseconds of thought I put into it probably wasn't enough :) . I can change it to BUILD_VARIANT.

I'm hoping this won't be a long lived variable name, but I'm happy to change it to something you find more descriptive :) .